gui/packages/ubuntupro/lib/pages/subscribe_now/subscribe_now_page.dart

```diff
@@ -21,12 +23,23 @@ class SubscribeNowPage extends StatelessWidget {
     final model = context.watch<SubscribeNowModel>();
     final lang = AppLocalizations.of(context);
     final theme = Theme.of(context);
+    final linkStyle = MarkdownStyleSheet.fromTheme(
+      theme.copyWith(
+        textTheme: theme.textTheme.copyWith(
+          bodyMedium: theme.textTheme.bodyLarge,
+        ),
+      ),
+    );
+
     return LandingPage(
       children: [
-        Text(
-          lang.proHeading,
-          style:
-              theme.textTheme.bodyLarge!.copyWith(fontWeight: FontWeight.w100),
+        SizedBox(
+          width: 400,
+          child: MarkdownBody(
+            data: lang.proHeading('[Learn more](https://ubuntu.com/pro)'),
```

Comment:
"Learn more" cannot be localized this way, but it should be. I think a better way to solve this problem is by having it as a separate l10n string.

```dart
data: lang.proHeading('[${lang.learnMore}](https://ubuntu.com/pro)'),
```

It's a good idea to avoid placing the markdown syntax directly in the localizable strings, as a translator could accidentally break the link for a specific language.